eth/gasprice/gasprice.go

```diff
@@ -29,7 +29,9 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
-const sampleNumber = 3 // Number of transactions sampled in a block
+// Number of transactions sampled in a block
+// Roughly the last 5 minutes
```

Comment:
`sampleNumber` refers to the transactions sampled in a single block. It's not relevant with time interval. Please correct the comment.

Reply:
I actually misinterpreted this variable, I assumed it was the number of blocks being sampled. Will update, would it be overkill if I were to increase this a little bit further for better sampling?

Reply:
I don't think this needs any increase. The intention is not to find the medium price, but rather "what were the cheapest transactions that successfully made it into this block?". 1 is a bit too low, but 3 seems fine to me.